app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -35,39 +34,24 @@ public LoginActivitySteps(LoginActivity activityClass) {
     }
 
     @Given("^I have a LoginActivity")
-    public void I_have_a_LoginActivity() {
+    public void i_have_a_LoginActivity() {
```

Comment:
In that case we'll have to update the Checkstyle ruleset :wink: You can find more info about the relevant rule [here](http://checkstyle.sourceforge.net/config_naming.html#MethodName)